### PR TITLE
Master sale generate link arj

### PR DESCRIPTION
**Pull request by arj-odoo**
[IMP] web: CopyClipboardCharLong widget for long strings
    
     - The copy CopyClipboardChar widget is not nicely rendered for very long strings (e.g. URL).
     - This commit modify the behavior of the CopyClipboardChar widget to display it nicely win such cases.
    
TaskID: 2059972

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by arj-odoo** [comment]
@VincentSchippefilt I modified the code to modify the default behavior of the CopyClipboardChar widget. I did not observed any bug in other modules where these widgets are used (CopyClipboardChar and CopyClipboardtext).

File: addons/payment/wizards/payment_link_wizard_views.xml [hunk 1]
```diff
@@ -6,14 +6,16 @@
         <field name="arch" type="xml">
             <form string="Generate Payment Link">
                 <group>
```
Comment (VincentSchippefilt) [inline addons/payment/wizards/payment_link_wizard_views.xml]: why does this change belong to this PR ?

Comment (arj-odoo) [reply addons/payment/wizards/payment_link_wizard_views.xml]: At the origin the task was related to the widget in this wizard. I will remove it from the PR.

Comment (VincentSchippefilt) [reply addons/payment/wizards/payment_link_wizard_views.xml]: maybe make 2 commits then, they may belong together in the PR but should be merged separately

Comment (arj-odoo) [reply addons/payment/wizards/payment_link_wizard_views.xml]: Done

**Comment by VincentSchippefilt** [comment]
robodoo rebase-ff

**Comment by dbo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**